packs/data/feats.db/shory-aerialist.json

```diff
@@ -29,7 +30,25 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": {
+                    "all": [
+                        "action:maneuver-in-flight"
+                    ]
+                },
+                "selector": "acrobatics",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "fly-speed",
+                "type": "status",
+                "value": 5
```

Comment:
This is technically wrong. We don't have a way to denote that you are flying via magic or because you have something else like an inventor item or some versatile heritage nonsense.

Reply:
I struggled with this one. Is there a non-magical way to fly? I think this bonus will be correct in 99% of cases, but can remove it if we need to go for 100%

Reply:
Move speeds are a mess, but yeah there's non magical flight. The Soaring Dynamo feat or Blast Boots for example. But yeah it would be right for most characters and the ones it isn't right for are unlikely to take the feat so it's maybe fine? Maybe we could have some clever way to parse the traits of the thing granting a move speed in the future? Though that seems a bit excessive for tackling one feat like this.